.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 references:
   container_config: &container_config
     docker:
-      - image: electronuserland/builder:wine-chrome-05.18
+      - image: electronuserland/builder:wine-chrome-05.21
```

Comment:
Have you verified that this actually works?

Reply:
See this https://github.com/CityOfZion/neon-wallet/pull/2227#issuecomment-913486861
without this change the windows build/deploy does not work.